hoot-core-test/src/test/cpp/hoot/core/algorithms/RdpWayGeneralizerTest.cpp

```diff
@@ -69,10 +62,10 @@ class RdpWayGeneralizerTest : public CppUnit::TestFixture
 
 public:
 
-  QMap<QString, QList<ConstNodePtr > > _inputPointsCache;
-  QMap<QString, QList<Coordinate> > _inputCoordsCache;
+  QMap<QString, QList<ConstNodePtr>> _inputPointsCache;
+  QMap<QString, QList<Coordinate>> _inputCoordsCache;
```

Comment:
One of my favorite new things with C++11, the `>>` at the end of the double template isn't read as the insertion operator anymore.  Someone was a genius on the parser team for GCC.